pypeit/traceslits.py

```diff
@@ -1,57 +1,59 @@
 """
 Module for guiding Slit/Order tracing
+
+.. _numpy.ndarray: https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.html
+
 """
+import os
 import inspect
 import copy
+
 import numpy as np
-import os
-from subprocess import Popen
 
 from scipy import ndimage
 
-from importlib import reload
-
 from astropy.io import fits
 
+# TODO: I'd rather we not use 'as' ...
```

Comment:
ok, but how does one deal with multiple instances of `utils`, `io`, etc.?

Reply:
It's actually not used in this file, so I'm just going to remove it.

I would generally just do: `import linetools.utils` in cases where there are multiple `utils` modules to load.  But I realize this makes the subsequent calls more onerous.